apps/docs/content/guides/functions/import-maps.mdx

```diff
@@ -8,59 +8,45 @@ tocVideo: 'ILr3cneZuFk'
 
 Developing with Edge Functions is similar to developing with Node.js, but with a few key differences. This guide will help you understand how to manage your dependencies.
 
-## Importing dependencies
+## Managing dependencies
 
-Supabase Edge Functions support:
+There are two ways to manage your dependencies in Supabase Edge Functions:
 
-- The Deno [standard library](https://deno.land/std)
-- JavaScript modules from npm (https://docs.deno.com/examples/npm/)
-- Built-in [Node APIs](https://docs.deno.com/runtime/manual/node/compatibility)
-- Third party modules published to [JSR](https://jsr.io/) or [deno.land/x](https://deno.land/x)
+### Using deno.json (Recommended)
```

Comment:
Might be worth mentioning the minimum CLI version users should use to make use of this `deno.json`

Reply:
Now that it's released and I have the version I added the info.